src/processors/universal.ts

```diff
@@ -34,8 +35,11 @@ class UniversalProcessor extends BaseOrderProcessor {
   private priorityQueue = new Set<string>();
   private queue = new Set<string>();
   private ordersMap = new Map<string, IncomingOrderContext>();
-
+  private readonly UNLOCK_BATCH_SIZE = 10;
   private isLocked: boolean = false;
+  private isUnlockBatchLocked: boolean = false;
+  private unlockBatchesOrderIdMap = new Map<string, Set<string>>();
```

Comment:
Use comments to clarify what's inside of this map. Looking at this line of code I can't understand, what stands for `string`, and what stands for `Set<string>`?